tests/microos/networkmanager.pm

```diff
@@ -55,7 +64,7 @@ sub run {
         die "$pkg will not be installed by default on SLE Micro" if (script_run("rpm -q $pkg") == 0);
     }
     my ($RcManager, $mode);
-    $mac_addr = get_var('NICMAC');
+    $mac_addr = is_s390x ? get_var('VIRSH_MAC') : get_var('NICMAC');
```

Comment:
IMHO it would be better to group the test data in a hash

Reply:
will do